worker/ffmpeg-worker-hpa.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: ffmpeg-worker
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: ffmpeg-worker
+  metrics:
+  - type: Resource
+    resource:
+      name: cpu
+      target:
+        type: Value
+        averageValue: 1000
```

Comment:
Can you add a comment here, on what are the units of cpu . i.e. what does "1000" mean?